70/EmailPage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+
+
+namespace _70
+{
```

Comment:
Format your code in this file: add access modifiers for class and IWebDriver, add spaces between methods